@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/event/fn_eventFired.sqf

```diff
@@ -21,7 +21,8 @@ Author:
 ---------------------------------------------------------------------------- */
 params["_heli", "_weapon", "_muzzle", "_mode", "_ammotype", "_magazine", "_missobj"];
 
-if !(local _heli) exitwith {};
+_gunner = getShotParents _missobj#1;//EH Gunner PARAM is fucked
```

Comment:
Can you make this `private`, also could you reword that comment to 'unable to retrieve correct gunner from EH`